### DISCUSSION TITLE
Is it any file format needed for pyreadstat.read_dta() ?

### DISCUSSION THREAD

**Discussion by hehuanxiang** [Roche/pyreadstat#121] [General]
Hello engineer, I am a new pythoner.
recently, I am using pyreadstat.read_dta to read my data, the data is like this:
![image](https://user-images.githubusercontent.com/48440077/113963301-ec5e9700-985b-11eb-9e8f-5ea38563e7ea.png)
![image](https://user-images.githubusercontent.com/48440077/113963347-00a29400-985c-11eb-9a3d-4d761d6d096c.png)
when I read the data, it gets error
![image](https://user-images.githubusercontent.com/48440077/113963416-20d25300-985c-11eb-9dd7-85e64c7fe09b.png)
I really hope that you can help me to get the solution
thank you so much

**Comment by ofajardo** [comment]
Please see  my answer [here](https://github.com/Roche/pyreadstat/issues/122)
